manifests/a/Albion/Online/20230307102657/Albion.Online.installer.yaml

```
@@ -0,0 +1,13 @@
+  PackageIdentifier: Albion.Online
+  PackageVersion: "20221214104247"
```

The `PackageVersion` of the installer should match the updated version number.
```suggestion
  PackageVersion: 20230307102657
```